editors/code/package.json

```diff
@@ -655,24 +655,30 @@
         "problemMatchers": [
             {
                 "name": "rustc",
+                "owner": "rustc",
```

Comment:
What's the best way to indicate that these values need to be kept in sync with what the language server is outputting?

Reply:
There's no good way, this'll go to https://github.com/rust-lang/vscode-rust soon. This is one of the things which are just easier to fix when bugs are discovered, than to protect against :)